diff -Nur nvidia-drivers-21.2.0.orig/set-ld-library-path.sh nvidia-drivers-21.2.0/set-ld-library-path.sh
--- nvidia-drivers-21.2.0.orig/set-ld-library-path.sh	1970-01-01 01:00:00.000000000 +0100
+++ nvidia-drivers-21.2.0/set-ld-library-path.sh	2015-03-05 12:19:00.235541451 +0100
@@ -0,0 +1,6 @@
+if [ -z "$LD_LIBRARY_PATH" ]; then
+    LD_LIBRARY_PATH=/usr/lib/tegra:/usr/lib/tegra-egl
+else
+    LD_LIBRARY_PATH="/usr/lib/tegra:/usr/lib/tegra-egl:$LD_LIBRARY_PATH"
+fi
+export LD_LIBRARY_PATH
